linkerd/app/integration/src/tests/identity.rs

```diff
@@ -25,7 +25,7 @@ async fn nonblocking_identity_detection() {
         .await;
     let proxy = proxy::new().identity(id_svc);
 
-    let msg1 = "custom tcp hello";
+    let msg1 = "custom tcp hello\n";
```

Comment:
nit/TIOLI: do you think it's worth also having some tests with `\r\n`s in them, just to make sure we're handling that correctly? the logic isn't too complex so i'm not *too* concerned about it, just a thought...

Reply:
the http detect tests have specific tests for this sort of thing. i think the integration tests should be as simple as possible to test whatever we expect them to tests.